src/Raven.Server/Documents/AI/ChatCompletionClient.cs

```diff
@@ -41,6 +41,7 @@ internal class ChatCompletionClient : IChatCompletionClient, IChatCompletionClie
     private readonly string _organizationId;
     private readonly string _projectId;
     private readonly bool? _think;
+    private readonly double? _temperature;
```

Comment:
I would made a class out of this something like `ExecutionSettings`.. there are many toggles here.
we can also can try to use the `Seed` one (if exists in the ollama)